lib/languages/js/source-code/source-code.js

```diff
@@ -1303,7 +1303,7 @@ class SourceCode extends TokenStore {
 					new VisitNodeStep({
 						target: node,
 						phase: 1,
-						args: [node, node.parent],
+						args: [node],
```

Comment:
Technically, this is breaking change since [`SourceCode` is public API](https://github.com/eslint/eslint/blob/61778f6ca33c0f63962a91d6a75a4fa5db9f47d2/lib/api.js#L49) and `traverse()` is a public method, though it's unlikely that it is used by anyone aside from ESLint core.

Reply:
It would still satisfy the interface required by ESLint, but for other direct consumers of this method this would be a breaking change in behavior because this particular implementation was returning the parent as `args[1]`.

If we don't think there are any other direct consumers of this method (i.e., only ESLint is using `SourceCode#traverse()`), then it's fine.

Reply:
FWIW typescript-eslint doesn't use it. I've searched around to try to find other consumers and the only usage I could find after a few minutes was [TSSLint, which passes no parameters](https://sourcegraph.com/github.com/johnsoncodehk/tsslint/-/blob/packages/eslint/index.ts?L552).